src/client/assets/styles/editor.scss

```diff
@@ -3,6 +3,7 @@
 @import 'fabric';
 @import 'components';
 @import 'containers';
+@import 'theme';
 
 .app {
     view-mode-error {
```

Comment:
Are "view-mode-error" and "view-mode"  angular components?  And so that's why they're not prefaced with a "." or a "#"?  Just want to make sure I understand this...